dotnet/src/SemanticKernel.Skills/Skills.Web/WebFileDownloadSkill.cs

```diff
@@ -73,7 +74,7 @@ public async Task DownloadToFileAsync(string source, SKContext context)
         this._logger.LogDebug("Response received: {0}", response.StatusCode);
 
         using Stream webStream = await response.Content.ReadAsStreamAsync();
-        using FileStream outputFileStream = new FileStream(filePath, FileMode.Create);
+        using FileStream outputFileStream = new FileStream(Environment.ExpandEnvironmentVariables(filePath), FileMode.Create);
```

Comment:
Would be nice to add some unit or integration tests for this.

Reply:
@craigomatic could you track this for the future pls, and resolve the comment? I think we can wait a few days but I'd rather not forget.

Reply:
I already did write some - Looks like they got lost during a merge with the changes that changed a bunch of directpry names.  I'll add this back shortly and will let you know when I do.

Reply:
Actually, the integration test already lives in "..\semantic-kernel\dotnet\src\SemanticKernel.IntegrationTests\WebSkill\WebSkillTests.cs" (WebFileDownloadSkillFileTestAsync) so we are good!

This conversation can be closed.